.docker/docker-compose.production.yml

```diff
@@ -2,33 +2,44 @@ version: "3.5"
 services:
   tdctl_api:
     image: ghcr.io/td-org-uit-no/tdctl-api/tdctl-api:latest
+    container_name: tdctl-api-production
     build:
       context: ../
       dockerfile: .docker/dockerfile.api.production
     volumes:
-      - ../db/seeds:/var/www/db/seeds
-      - ../db/eventImages:/var/www/db/eventImages
-      - ../db/eventExports:/var/www/db/eventExports
-        # mount .config as mail credentials cannot be on github
-      - ../.config:/var/www/.config
+      - ../db/seeds:/home/prod_user/db/seeds
+      - ../db/eventImages:/home/prod_user/db/eventImages
+      - ../db/eventExports:/home/prod_user/db/eventExports
+      # mount .config as mail credentials cannot be on github
+      - ../.config:/home/prod_user/.config
     environment:
-      FLASK_APP_ENV: production
-      DB_HOSTNAME: mongodb
-      DB_PORT: 27017
+      API_ENV: production
+      DB_USER: ${MONGODB_USER}
+      DB_PASSWORD: ${MONGODB_PASSWORD}
+      DB_HOSTNAME: ${HOSTNAME}
+      DB_PORT: ${PORT}
       SECRET_KEY: thisisverysecret
```

Comment:
Is this used? Should maybe either remove it or change it out with an env variable like the others (also change the secret key and not reuse `thisisverysecret` since it's already in plaintext in the git log :sweat_smile: )

Reply:
yes, i don't think it is used, kinda inherit code, I guess. I think I'll just remove it :fire: